strongbox-web-core/pom.xml

```diff
@@ -508,4 +511,73 @@
 
+    <profiles>
+        <profile>
+            <id>jdk-default</id>
+            <activation>
+                <jdk>!11</jdk>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.springframework.boot</groupId>
+                        <artifactId>spring-boot-maven-plugin</artifactId>
+                        <configuration>
+                            <jvmArguments>
+                                -Xmx1024m
+                            </jvmArguments>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>jdk11</id>
+            <activation>
+                <jdk>11</jdk>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.springframework.boot</groupId>
+                        <artifactId>spring-boot-maven-plugin</artifactId>
+                        <configuration>
+                            <jvmArguments>
+                                -Xmx1024m
+                                -Djdk.attach.allowAttachSelf=true
+                                --add-exports java.base/jdk.internal.misc=ALL-UNNAMED
+                                --add-exports java.base/jdk.internal.ref=ALL-UNNAMED
+                                --add-exports java.base/sun.nio.ch=ALL-UNNAMED
+                                --add-exports
+                                java.management.rmi/com.sun.jmx.remote.internal.rmi=ALL-UNNAMED
+                                --add-exports
+                                java.rmi/sun.rmi.registry=ALL-UNNAMED
+                                --add-exports
+                                java.rmi/sun.rmi.server=ALL-UNNAMED
+                                --add-exports
+                                java.sql/java.sql=ALL-UNNAMED
+
+                                --add-opens
+                                java.base/java.lang.module=ALL-UNNAMED
+                                --add-opens
+                                java.base/jdk.internal.loader=ALL-UNNAMED
+                                --add-opens
+                                java.base/jdk.internal.ref=ALL-UNNAMED
+                                --add-opens
+                                java.base/jdk.internal.reflect=ALL-UNNAMED
+                                --add-opens
+                                java.base/jdk.internal.math=ALL-UNNAMED
+                                --add-opens
+                                java.base/jdk.internal.module=ALL-UNNAMED
+                                --add-opens
+                                java.base/jdk.internal.util.jar=ALL-UNNAMED
+                                --add-opens
+                                jdk.management/com.sun.management.internal=ALL-UNNAMED
+                            </jvmArguments>
```

Comment:
Cosmetic: Could you please make the `--add-exports` and `--add-opens` one-liners like this:
```suggestion
                            <jvmArguments>
                                -Xmx1024m
                                -Djdk.attach.allowAttachSelf=true
                                --add-exports java.base/jdk.internal.misc=ALL-UNNAMED
                                --add-exports java.base/jdk.internal.ref=ALL-UNNAMED
                                --add-exports java.base/sun.nio.ch=ALL-UNNAMED
                                --add-exports java.management.rmi/com.sun.jmx.remote.internal.rmi=ALL-UNNAMED
                                --add-exports java.rmi/sun.rmi.registry=ALL-UNNAMED
                                --add-exports java.rmi/sun.rmi.server=ALL-UNNAMED
                                --add-exports java.sql/java.sql=ALL-UNNAMED

                                --add-opens java.base/java.lang.module=ALL-UNNAMED
                                --add-opens java.base/jdk.internal.loader=ALL-UNNAMED
                                --add-opens java.base/jdk.internal.ref=ALL-UNNAMED
                                --add-opens java.base/jdk.internal.reflect=ALL-UNNAMED
                                --add-opens java.base/jdk.internal.math=ALL-UNNAMED
                                --add-opens java.base/jdk.internal.module=ALL-UNNAMED
                                --add-opens java.base/jdk.internal.util.jar=ALL-UNNAMED
                                --add-opens jdk.management/com.sun.management.internal=ALL-UNNAMED
                            </jvmArguments>
```

Thanks! :)